client/src/browser_action/react-contextmenu.css

```diff
@@ -17,6 +17,8 @@
 .react-contextmenu.react-contextmenu--visible {
     opacity: 1;
     pointer-events: auto;
+    position: absolute;
```

Comment:
I just checked, react-contextmenu adds position: fixed so your position: absolute is overwritten